subprojects/diagnostics/src/integTest/groovy/org/gradle/api/reporting/dependencies/HtmlDependencyReportTaskIntegrationTest.groovy

```diff
@@ -563,14 +563,15 @@ rootProject.name = 'root'
                maven { url "${mavenRepo.uri}" }
             }
             configurations {
-                createWithRole('compileOnly', org.gradle.api.internal.artifacts.configurations.ConfigurationRolesForMigration.RESOLVABLE_BUCKET_TO_BUCKET)
+                migratingUnlocked('compileOnly', org.gradle.api.internal.artifacts.configurations.ConfigurationRolesForMigration.LEGACY_TO_CONSUMABLE)
```

Comment:
Yes. This test only needed a configuration that was deprecated for resolution. `RESOLVABLE_BUCKET_TO_BUCKET` was otherwise unused in tests, so I removed its usage in the test and switched it out for another role which was acceptable.